index.bs

```diff
@@ -3085,96 +3097,105 @@ partial interface MLGraphBuilder {
     The behavior of this operation can be [EMULATED]
   </summary>
   <pre highlight="js">
-    function squeeze(builder, op) {
-      return builder.reshape(op, op.shape().remove(0));
-    }
-
-    const numDirections = (options.direction == 'both' ? 2 : 1);
-    let hiddenState = options.initialHiddenState;
-
-    if (!hiddenState) {
-      const desc = {
-        dataType: 'float32',
-        dimensions: [numDirections, 1, hiddenSize]
-      };
-      const totalSize = numDirections * hiddenSize;
-      hiddenState = builder.constant(desc, new Float32Array(totalSize).fill(0));
-    }
+    function gru(
+      builder, input, weight, recurrentWeight, steps, hiddenSize, options) {
+      function squeeze(builder, op) {
```

Comment:
Should we just use https://www.w3.org/TR/webnn/#emulation-squeeze?

Reply:
I could go either way. I like it being standalone, but there's some benefit in avoiding repetition. Linking to the squeeze emulation from the code might be awkward.

Reply:
Actually... these inline `function squeeze` examples use `op.shape().remove(0)` which was introduced in https://github.com/webmachinelearning/webnn/pull/478/commits/e284f2acc4c8db0db5f2415adc8be7ef188299c6 ... I'm not sure what the `remove()` is intending to do as there is no such method on Array.

Reply:
Okay, replaced with a link in 73bca91 - I think this is better.